controllers/datadogagent/feature/clusterchecks/feature.go

```diff
@@ -57,22 +57,21 @@ func (f *clusterChecksFeature) ID() feature.IDType {
 	return feature.ClusterChecksIDType
 }
 
-func (f *clusterChecksFeature) Configure(dda *v2alpha1.DatadogAgent) feature.RequiredComponents {
-	clusterChecksEnabled := apiutils.BoolValue(dda.Spec.Features.ClusterChecks.Enabled)
-	f.useClusterCheckRunners = clusterChecksEnabled && apiutils.BoolValue(dda.Spec.Features.ClusterChecks.UseClusterChecksRunners)
-
-	{
-		hash, err := comparison.GenerateMD5ForSpec(dda.Spec.Features.ClusterChecks)
-		if err != nil {
-			f.logger.Error(err, "couldn't generate hash for cluster checks config")
-		} else {
-			f.logger.V(2).Info("created cluster checks", "hash", hash)
+func (f *clusterChecksFeature) Configure(dda *v2alpha1.DatadogAgent) (reqComp feature.RequiredComponents) {
+	if apiutils.BoolValue(dda.Spec.Features.ClusterChecks.Enabled) {
+		f.useClusterCheckRunners = apiutils.BoolValue(dda.Spec.Features.ClusterChecks.UseClusterChecksRunners)
+
+		{
```

Comment:
i think this set of brackets is not needed? although curiously i see it was there before

Reply:
Yes, it's not really needed. Not sure what original intent was but it improves readability and narrows variable scope. I moved it to a method.